draft.md

```diff
@@ -0,0 +1,101 @@
+# Draft IPLD Unixfs Spec
+
+## Basic Structure
+
+A Unixfs is either a file or a directory.
+The top level IPLD object is a CBOR map with at least two fields: `type` and `data`
+and maybe a few other such as a version string or a set of flags.
+The `type` field is either `file` or `dir`.
```

Comment:
I say better to define a CBOR tag for files and and a tag directories, and define a *file* as a tagged array and a *dir* as a tagged map. That makes it clear from the first atomic in the CBOR that you are parsing UnixFS.

Reply:
@ehmry we can do that, but do we then need to register the tags?

Reply:
Not really, I was thinking just picking two random uint64 tags.